pilot/pkg/networking/core/v1alpha3/envoyfilter/listener_patch_test.go

```diff
@@ -1423,7 +1423,7 @@ func BenchmarkTelemetryV2Filters(b *testing.B) {
 								AlwaysSetRequestIdInResponse: true,
 								HttpFilters: []*http_conn.HttpFilter{
 									{Name: "http-filter3"},
-									{Name: "envoy.router"}, // Use deprecated name for test.
+									{Name: "envoy.filters.http.router"}, // Use deprecated name for test.
```

Comment:
This is left intentionally with the deprecated name for the test to verify that listener patch works with both deprecated and new canonical names

Reply:
Will revert